backend/src/main/java/gov/cdc/usds/simplereport/service/ResultsUploaderDeviceValidationService.java

```diff
@@ -53,6 +106,30 @@ public void cacheModelAndTestPerformedCodeToDeviceMap() {
     getModelAndTestPerformedCodeToDeviceMap();
   }
 
+  @Cacheable(SPECIMEN_NAME_AND_SNOMED_MAP)
+  public Map<String, String> getSpecimenTypeNameToSNOMEDMap() {
```

Comment:
I dithered on including this here or separating it out into its own service - happy to break this out if anyone would like.

Reply:
Its not that bad, specimens are part of the device